src/libs/ReportUtils.js

```diff
@@ -434,6 +450,23 @@ function canShowReportRecipientLocalTime(personalDetails, report) {
         && isReportParticipantValidated);
 }
 
+/**
+ * @param {Object} report
+ * @returns {String}
+ */
```

Comment:
Not sure how easy it is to make it compact but I feel like some summary of what this function does would be helpful here so developer knows at glance what will be the result here.

Reply:
done